services/home/src/JoinStudy/components/JoinStudyBox/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { Button, Image, Tag, Text } from '@depromeet/toks-components';
+import { Flex, Spacing, width100 } from '@toss/emotion-utils';
+
+import { StudyInfo } from 'JoinStudy/components/StudyInfo';
+import { StudyTitle } from 'JoinStudy/components/StudyTitle';
+
+import { Wrapper } from './style';
+
+export function JoinStudyBox() {
+  // mock data
+  const ourStudyDescription =
+    '설명은 100자 제한입니다 설명은 100자 제한입니다 설명은 100자 제한입니다 설명은 100자 제한입니다 설명은 100자 제한입니다 설명은 100자 제한입니다 설명은 100자 제한입니다 설명은 100자 제한입니다 설명은 100자 제한입니다 설명은 100자 제한';
+  const personnelDescription = '5-7명을 계획하고 있어요.';
+  const startDate = '2022. 10. 13';
+  const doneDate = '2022. 12. 03';
+  const tags = ['Java', 'Javascript', 'React'];
+  return (
+    <Wrapper>
+      <div>
+        <StudyTitle />
+        {/* tag margin 위아래 10 고려하여 18->8 */}
+        <Spacing size={8} />
+        <Tag.Row>
+          {tags?.map(tag => (
+            <Tag key={tag}>{tag}</Tag>
+          ))}
+        </Tag.Row>
+      </div>
+      <Flex direction="column" css={{ gap: '32px' }}>
+        <StudyInfo
+          leftAddon={
+            <Image
+              width={16}
+              height={22}
+              src="https://toks-web-assets.s3.amazonaws.com/studyinfo-icon.svg"
+              alt="StudyInfo 아이콘"
+            />
+          }
+          title={'우리 스터디는'}
+          description={<Text variant="body01">{ourStudyDescription}</Text>}
+        />
+        <StudyInfo
+          leftAddon={
+            <Image
+              width={16}
+              height={22}
+              src="https://toks-web-assets.s3.amazonaws.com/studyinfo-icon.svg"
+              alt="StudyInfo 아이콘"
+            />
+          }
+          title={'스터디 기간은'}
+          description={
+            <>
+              <Text variant="body02">시작일</Text>
+              <Text variant="body01" css={{ marginLeft: '12px' }}>
+                {startDate}
+              </Text>
+              <Text variant="body02" css={{ marginLeft: '36px' }}>
+                종료일
+              </Text>
+              <Text variant="body01" css={{ marginLeft: '12px' }}>
+                {doneDate}
+              </Text>
```

Comment:
```suggestion
              <Text variant="body01" css={{ marginLeft: '12px' }}>
                {startDate}
              </Text>
              <Text variant="body02" css={{ marginLeft: '36px' }}>
                종료일
              </Text>
              <Text variant="body01" css={{ marginLeft: '12px' }}>
                {doneDate}
              </Text>
```

요거는 캘린더가 들어가는걸로 아는데 텍스트말고 다른거로 바꿔주실 수 있나요? 커스텀은 나중에하더라도 날짜 선택 기능이 된다면 좋을 것 같아요

Reply:
어랏 여기 캘린더가 들어가나요?! 스터티 생성에서는 캘린더가 들어가는 것 같은데 여기는 링크로 참여하는 페이지여서 생성시에 정해진 날짜를 스트링으로 보여줘도 괜찮지 않을까요?!?! 훔

Reply:
허허 뻘소리했네요 ^ㅅ^ 맞습니다..